### PR TITLE
Added command for finding students by their Class Code

### LINKED ISSUE #8: As a User, I can filter students by their Class Code
As a user, I can filter students by their Classes to see the state of each class, and how many Covid-19 Cases are there in each class.

### PR DESCRIPTION
**Pull request by Fenway17**
Changes made:
- new class FindClassCodeCommand.java
- new class FindClassCodeCommandParser.java
- new class ClassCodeContainsKeywordsPredicate.java
- edited AddressBookParser.java to accommodate new Command

Removed:
POTENTIAL_CHANGES.txt used in v1.1

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/FindClassCodeCommand.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import seedu.address.model.Model;
+import seedu.address.model.person.ClassCodeContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose class code matches the given class code.
+ * Keyword matching is case insensitive.
+ */
+public class FindClassCodeCommand extends Command {
+
+    public static final String COMMAND_WORD = "findclasscode";
```
Comment (louisdavinlie) [inline src/main/java/seedu/address/logic/commands/FindClassCodeCommand.java:15]: Should the COMMAND_WORD be find_by_class to follow our user guide?
```suggestion
    public static final String COMMAND_WORD = "find_by_class";
```

**Review by louisdavinlie — COMMENTED** [review]
Looks good. Just a minor doubt on name of command.

**Review by joshuayapwj98 — COMMENTED** [review]
Looks good! Louis already pointed out a potential change for the command word.

Otherwise, approved from my end.

File: src/main/java/seedu/address/logic/commands/FindClassCodeCommand.java (hunk 1 above)
Comment (whoisjunhong) [reply src/main/java/seedu/address/logic/commands/FindClassCodeCommand.java:15]: I would recommend sticking with the previous convention as the user guide is subjected to change

**Comment by whoisjunhong** [comment]
LGTM.

**Pull request merged**